### ISSUE TITLE
Multiline HTML is opinionated

### ISSUE THREAD

**Issue by Thutmose3** [rtts/djhtml#82]
Hi,

I have been looking for a long time for something like this, but they never worked as expected.
I think one of the reasons is that there is no standard way to write Django HTML.

The solution would be to have a opinionated stance on how to format Django HTML (like Black does with python). Anyway, thank you for this library. I'll try my best to help with this project.

So the issue i'm facing is this:

Initial code:
```
<ul>
    <li class="breadcrumb-item text-muted">
        <a
        href="{% url 'dashboard' %}"
        class="text-muted text-hover-primary"
        hx-target="#main_content"
        hx-swap="innerHTML show:#kt_app_root:top"
        >
            {% trans "Home" %}
        </a>

        <button
        class="btn btn-sm btn-success"
        hx-get="{% url 'get_brand_form' %}"
        hx-target="#generic_modal_form_placeholder"
        >
            {% trans "Create" %}
        </button>
    </li>
</ul>
```

expected result: 
```
<ul>
    <li class="breadcrumb-item text-muted">
        <a
            href="{% url 'dashboard' %}"
            class="text-muted text-hover-primary"
            hx-target="#main_content"
            hx-swap="innerHTML show:#kt_app_root:top"
        >
            {% trans "Home" %}
        </a>

        <button
            class="btn btn-sm btn-success"
            hx-get="{% url 'get_brand_form' %}"
            hx-target="#generic_modal_form_placeholder"
        >
            {% trans "Create" %}
        </button>
    </li>
</ul>
```

current result:
```
<ul>
    <li class="breadcrumb-item text-muted">
        <a
           href="{% url 'dashboard' %}"
           class="text-muted text-hover-primary"
           hx-target="#main_content"
           hx-swap="innerHTML show:#kt_app_root:top"
        >
            {% trans "Home" %}
        </a>

        <button
                class="btn btn-sm btn-success"
                hx-get="{% url 'get_brand_form' %}"
                hx-target="#generic_modal_form_placeholder"
        >
            {% trans "Create" %}
        </button>
    </li>
</ul>
```

as you can see the html attributes of the <a> have move 3 spaces to the right. While the attributes of <button> have moved 8 spaces to the right.

Using v3.0.1

**Comment by JaapJoris** [comment]
Thank you for taking the time to reach out! Also see the discussion at #50 where the following examples appear.

DjHTML version 2 and lower uses the following indentation:

```html
<img src="/some/long/path"
     id="1">
<blockquote class="dishy"
     id="2" />
<a href="/some/long/path"
     id="3"></a>
```

The latest version has changed that to this:

```html
<img src="/some/long/path"
     id="1">
<blockquote class="dishy"
            id="2" />
<a href="/some/long/path"
   id="3"></a>
```

In other words, attribute names are now always lined up with each other, which is also the way that many IDEs indent multi-line HTML elements. I agree on your stance that this looks inconsistent, but since this was the most often requested improvement we have changed it nonetheless.

In conclusion: this is the intended behavior, and DjHTML's opinionated stance is that attribute names should always line up perfectly with each other.

**Comment by Thutmose3** [comment]
Ah, I see clearly now!

It's not how I'm used to writing my Django HTML code. 
But if this is the norm, then I will change my habits.

Thanks!

**Comment by Thutmose3** [comment]
Although I don't find it really clean to have HTML code not indented with 4 spaces, the attributes in the `<a>` tag are currently indented 3 spaces, which is not very pythonic, and looks ugly and a bit wrong (python instincts probably).

I'm used to writing my HTML multiline code like this:
```
<a
    href="{% url 'dashboard' %}"
    class="text-muted text-hover-primary"
    hx-target="#main_content"
    hx-swap="innerHTML show:#kt_app_root:top"
>
    {% trans "Home" %}
</a>
```

Here the attributes are aligned under each other, they are indented with 4 spaces. and the attributes are formatted like python function arguments:

```
def update_outgoing_invoice(
    invoice=1,
    invoice_id="15615",
    invoice_date=None,
    language=None,
    extra_info=None,
    percentage_invoiced=None,
):
```

Overall this is a more pythonic/clean way to write HTML templates.

It would be nice when someone uses multiline HTML attributes, **without placing an attribute on the same level as the HTML element**, it indents in the way I described above (4 spaces)?

Can this feature be added?

**Comment by JaapJoris** [comment]
> without placing an attribute on the same level as the HTML element

This is a very good suggestion, and I will leave this issue open until I have gathered some more opinions on this proposed change!

**Comment by JaapJoris** [comment]
After careful consideration, I've concluded that your suggestion really makes a lot of sense and will also appease the existing users of DjHTML who were already used to this behavior before the fix for #50 was introduced in DjHTML 3. I actually consider it a bug that I didn't implement it this way in the first place, 

Thank you very much for suggesting this improvement. You should consider yourself special because from now on, I don't plan to change any more indentation rules (other than fixing obvious bugs). DjHTML should be like Black: stable and immutable, so our users can rely on a consistent set of indentation rules for their source code.

**Comment by Thutmose3** [comment]
I just ran this new version on my codebase, and it works flawlessly, thanks a lot!
